docs/canonicalk8s/custom_conf.py

```diff
@@ -137,7 +137,10 @@
 
 # Links to ignore when checking links
 linkcheck_ignore = [
-    'http://127.0.0.1:8000'
+    'http://127.0.0.1:8000',
+    'http://rocks.canonical.com',
+    'about',
+    'https://ceph.io/'
```

Comment:
why do we ignore the ceph link?

Reply:
The ceph.io and ceph.com website seems to be returning an error on the linkcheck when I run it. However, when I go from the docs it resolves fine. I am not sure why it is raising a false positive with the ceph site.